Package.swift

```diff
@@ -103,7 +103,7 @@ let package = Package(
         ),
         .target(
             name: "SmithyReadWrite",
-            dependencies: ["SmithyTimestamps"]
+            dependencies: ["Smithy", "SmithyTimestamps"]
```

Comment:
Since ByteStream is now in the `Smithy` module, `SmithyReadWrite` takes a dependency on `Smithy` and provides its own read-write implementation for `ByteStream`.